frontend/src/reducer/workspaceList.js

```diff
@@ -105,6 +105,14 @@ export function workspaceList (state = [], action, lang) {
       return sortListByMultipleCriteria(spaceList, [SORT_BY.LABEL, SORT_BY.ID], SORT_ORDER.ASCENDING, lang)
     }
 
+    case `${UPDATE}/${USER_WORKSPACE_EMAIL_NOTIFICATION_TYPE}`:
```

Comment:
I would also like a new const for this action

Reply:
done